python/GafferUI/SwitchUI.py

```diff
@@ -58,9 +58,23 @@
 				of 0 chooses the first input, 1 the second and so on. Values
 				larger than the number of available inputs wrap back around to
 				the beginning.
+				""",
+
+				"nodule:type", "",
+
+			],
+
+			"in" : [
+
+				"description",
 				"""
+				The array of inputs to choose from. One of these is chosen
+				by the index plug to be passed through to the output.
+				""",
+
+				"nodule:type", "GafferUI::CompoundNodule",
```

Comment:
Also the "in" array shouldn't have a widget in the NodeEditor. "in.*" and "out" should only have one if the plug they were setup from does.